benchmarks/dynamo/common.py

```diff
@@ -108,6 +108,7 @@
 current_onnx_compiler = ""
 current_batch_size = None
 output_filename = None
+disable_output = False
```

Comment:
I really don't love adding a global flag, but a) it will be a big change to thread this arg through all call-sites, and b) `output_filename` is already global and I'd at least prefer to keep it consistent.